### ISSUE TITLE
STA/OPR 9750 aaronkcooper MiniProject #02

### ISSUE THREAD

**Issue by aaronkcooper** [aaronkcooper/STA9750-2024-FALL#3]
Hi @michaelweylandt!

I've uploaded my work for MiniProject #02 - check it out!

https://aaronkcooper.github.io/STA9750-2024-FALL/mp02.html

**Comment by michaelweylandt** [comment]
Thanks @aaronkcooper !

Peer Feedback:

- @chricannon
- @YantingZhao11
- @Malexandra-Jerez
- @mussakone

It is now time for the peer feedback round for Mini-Project 02. Please
review @aaronkcooper's submission for this mini-project and provide peer feedback.

Using the rubric at https://michael-weylandt.com/STA9750/miniprojects/mini02.html#rubric,
please grade their submission out of a total of 50 points. Note that this
rubric is slightly different than that used for Mini-Project 01.

For each of the five categories, please give them a separate score and provide
a total (sum) score across the entire assignment. Feel free to assign extra
credit if you feel it is warranted (following the rubric).

If you give a score of less than 5 for any category, please provide a suggestion
for improvement. (You can also give suggestions for any element they did well -
more feedback is always great!)

As you go through this peer feedback exercise, think about what you particularly
like about this submission and how you can incorporate that approach in your future
work. If something is particularly insightful or creative, give some kudos!

Evaluators: This should take you around 15 minutes per peer feedback. You are
not required to engage in substantial back-and-forth with @aaronkcooper, but you are of course
welcome to initiate a discussion.

@aaronkcooper: please engage fully with your peers. They are here to help you!

Submission URL should be: https://aaronkcooper.github.io/STA9750-2024-FALL/mp02.html

Feel free to link to other repos, the course documentation, or other useful
examples.

Thanks!
@michaelweylandt

CC: @charles-ramirez

**Comment by Malexandra-Jerez** [comment]
Hi @aaronkcooper <https://github.com/aaronkcooper> !

Please see below review.

Written communication:  9, Great description, flows smoothly. You were able
to explain easily.

Project Skeleton: 10, All tasks completed and fully explained.

Formatting and Display: 9, The interactive tables were a plus, i was able
to nowrro down and filter through those. Although I would clean up the
tables a bit more to exclude columns not needed.

Code Quality: 10, clean, readable, well formatted.

Data preparation: 10


Kindly,
Alex

**Comment by Malexandra-Jerez** [comment]
Sum Total = 48

**Comment by mussakone** [comment]
Hi @aaronkcooper I have reviewed your project and here is what I think below:

Written Communication: 9/10
Your writing described all of your work clearly. You had captions with data to help us follow along. I would suggest your introduction be more captivating.

Project Skeleton: 10/10
All tasks of the project were completed. All task were neat and easy to follow.

Formatting and display:9/10
Your codes and tables were interactive and you allowed us search on our own. The graph were clear and accurate along with your coding. I would only suggest more color and maybe pictures to give your project an extra spin to it.

Code Quality: The coding was very clean and easy to following since you hid them and then we had to click into them. In addition, you didn't have any warnings within the code which is also a plus.

Data Prep: 10/10 All data is accurate

Overall: 48/50

**Comment by YantingZhao11** [comment]
HI @aaronkcooper,

Please see the below.

Written Communication: 8/10
Your writing was clear and easy to follow, with helpful data captions. A more engaging introduction and explanation of Task 2 could enhance it.

Project Structure: 10/10
All tasks were completed and well-organized, making the project easy to follow.

Formatting and Display: 9/10
Interactive tables and clear graphs added value. More color or visuals and removing unnecessary columns could improve readability.

Code Quality: 10/10
Code was clean, readable, and error-free, with hidden sections that expanded when needed.

Data Preparation: 10/10

Overall: 47/50

Thanks,
Yanting

**Comment by chricannon** [comment]
Hi @aaronkcooper - great job on MP 2! I enjoyed reviewing your project. 

Written Communication: 10/10
I appreciate your commentary throughout the projects. You made sure to comment throughout each task, which is something I did not see on each project I reviewed and I appreciate. 
I especially enjoyed reading through your Task 7 - The Ultimate Movie Pitch. Your last paragraph, with the trailer snippet was awesome. 

Project Skeleton: 10/10
Your project flows perfectly. All tasks are complete and easily identifiable. Nice! 

Formatting & Display: 9/10
Excellent use of creating interactive tables. Perhaps adding more color to your visuals and project throughout could be a slight improvement to make for the future. 

Code Quality: 10/10
Code is included for each section and runs - great job. I appreciate your use of code folding - this allows a reader to easily scroll through the project from start to finish on an initial view, without feeling bombarded with lots of code. 

Data Preparation: 10/10  (automatic) 
You properly loaded the data and engaged in the steps as outlined. One small improvement here could have been speaking to the data you are loading and details in what changes are made in the pre-processing/cleaning step. 

**Total: 49/50**
